pkgs/build-support/build-bazel-package/default.nix

```
@@ -50,13 +53,59 @@ let
   fArgs = removeAttrs args [ "buildAttrs" "fetchAttrs" "removeRulesCC" ];
   fBuildAttrs = fArgs // buildAttrs;
   fFetchAttrs = fArgs // removeAttrs fetchAttrs [ "sha256" ];
+  bazelCmd = { cmd, additionalFlags, targets }:
+    if targets == [ ] then "" else
+      ''
+          # Bazel sandboxes the execution of the tools it invokes, so even though we are
+          # calling the correct nix wrappers, the values of the environment variables
```

```suggestion
    lib.optionalString (targets != [ ]) ''
      # Bazel sandboxes the execution of the tools it invokes, so even though we are
      # calling the correct nix wrappers, the values of the environment variables
```
in nix files we use 2 spaces to intend